src/App.jsx

```diff
@@ -134,6 +135,94 @@ function App() {
                 each year without running out of money for at least 30 years. This calculation 
                 assumes your investments continue to grow during retirement.
               </p>
+              <button 
+                className="learn-more-btn"
+                onClick={() => setShowRuleModal(true)}
+              >
+                Learn More About the 4% Rule
+              </button>
+            </div>
+          </div>
+        )}
+
+        {showRuleModal && (
```

Comment:
⚖️ **Midlevel Developer**

Nice work on the ARIA labels (`aria-label="Close modal"`) for accessibility! However, for full accessibility coverage, you should also trap focus within the modal and return focus to the triggering element when it closes. At my last job, we used the `react-focus-lock` library for this. Consider adding at least one test to verify keyboard navigation and ESC key handling in the modal for a11y compliance.